WardenApi/src/main/java/com/salesforce/dva/warden/dto/Policy.java

```diff
@@ -1,5 +1,272 @@
+
 package com.salesforce.dva.warden.dto;
 
-public class Policy {
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Policy Dto.
+ *
+ * @author  Jigna Bhatt (jbhatt@salesforce.com)
+ */
+@SuppressWarnings("serial")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Policy extends Entity {
+	//~ Instance fields ******************************************************************************************************************************
+
+	    private String service;		
+	    private String name;
+	    private List<String> owners  = new ArrayList<String>();
+	    private List<String> users  = new ArrayList<String>();
+	    private String subSystem;
+	    private String metricName;
+	    private TriggerType triggerType;
+	    private Aggregator aggregator;
+	    private List<Double> threshold;	    
+	    private String timeUnit;
+	    private Double defaultValue;
+	    private String cronEntry;
+	    private List<BigInteger> suspensionLevelIds = new ArrayList<BigInteger>();
+
```

Comment:
@jbhatt-salesforce @saaja-sfdc , quick question, do we need infractionIds here? cause in policy entity we have  
    private List<Infraction> infractionList;